store/snapshots/helpers_test.go

```diff
@@ -236,39 +215,28 @@ func setupBusyManager(t *testing.T) *snapshots.Manager {
 	return mgr
 }
 
-// hungSnapshotter can be used to test operations in progress. Call close to end the snapshot.
-type hungSnapshotter struct {
-	ch               chan struct{}
-	prunedHeights    map[int64]struct{}
-	snapshotInterval uint64
+// hungCommitSnapshotter can be used to test operations in progress. Call close to end the snapshot.
+type hungCommitSnapshotter struct {
+	ch chan struct{}
 }
 
-func newHungSnapshotter() *hungSnapshotter {
-	return &hungSnapshotter{
-		ch:            make(chan struct{}),
-		prunedHeights: make(map[int64]struct{}),
+func newHungCommitSnapshotter() *hungCommitSnapshotter {
+	return &hungCommitSnapshotter{
+		ch: make(chan struct{}),
 	}
 }
 
-func (m *hungSnapshotter) Close() {
+func (m *hungCommitSnapshotter) Close() {
 	close(m.ch)
 }
 
-func (m *hungSnapshotter) Snapshot(height uint64, protoWriter protoio.Writer) error {
+func (m *hungCommitSnapshotter) Snapshot(height uint64, protoWriter protoio.Writer) error {
 	<-m.ch
 	return nil
 }
 
-func (m *hungSnapshotter) PruneSnapshotHeight(height int64) {
-	m.prunedHeights[height] = struct{}{}
-}
-
-func (m *hungSnapshotter) SetSnapshotInterval(snapshotInterval uint64) {
-	m.snapshotInterval = snapshotInterval
-}
-
-func (m *hungSnapshotter) Restore(
-	height uint64, format uint32, protoReader protoio.Reader,
+func (m *hungCommitSnapshotter) Restore(
+	height uint64, format uint32, protoReader protoio.Reader, chStorage chan<- *store.KVPair,
```

Comment:
Stylistic nit for unused functions parameters to even indicate to outside callers that it is not to be used
```go
func (m *hungCommitSnapshotter) Restore(_ uint64, _ uint32, _ protoio.Reader, _ chan<- *store.KVPair) (...) {
    panic("not implemented")
}
```